docs/DeveloperGuide.adoc

```diff
@@ -362,7 +362,7 @@ _{More to be added}_
 (For all use cases below, the *System* is the `Mark` and the *Actor* is the `user`, unless specified otherwise)
 
 [discrete]
-=== Use case 01: List bookmarks
+=== Use case UC01: List bookmarks
```

Comment:
Suggested change in format to match the textbook (and likewise for the other UCs):
```suggestion
=== Use case: UC01 - List bookmarks
```